core/solver/batch_bicgstab_kernels.hpp

```diff
@@ -88,8 +88,7 @@ void set_gmem_stride_bytes(storage_config& sconf,
         gmem_stride += prec_storage_bytes;
     }
     // align global memory chunks
-    sconf.gmem_stride_bytes =
-        gmem_stride > 0 ? ceildiv(gmem_stride, align_bytes) * align_bytes : 0;
+    sconf.gmem_stride_bytes = ceildiv(gmem_stride, align_bytes) * align_bytes;
```

Comment:
I don't think this is equivalent to the one before. Is that intentional?

Reply:
I think this was left over from previous code and not necessary anymore as ceildiv should be okay with gmem_stride being zero